### DISCUSSION TITLE
Practice Questions

### DISCUSSION THREAD

**Discussion by kostasim** [jspsych/jsPsych#2267] [Q&A] [answered]
Hello

I have seen that for practice questions the following  iat_type can be used for example: `data: { iat_type: 'practice' }`

In the score calculation this will not be used since the values for the calculation are filtered by the iat_type for example: `jsPsych.data.get().filter({iat_type: 'bad-old'})`

So the calculation is always dependent on the filter as I understood.

Just want to verify this :)

Thanks
Kosta

**Comment by jodeleeuw** [comment, marked as answer]
Without seeing more of the code I don't think I can conclusively verify this, but what you've described seems accurate.

If you run `jsPsych.data.get().filter({iat_type: 'bad-old'})` then you will get a `DataCollection` with only the trials that contain `data: {iat_type: 'bad-old'}`. 

If you want to double check this, you could use the `count()` method on the resulting `DataCollection` to check that the correct number of trials are in the collection.

**Reply by kostasim** [reply]
thanks for the answer. That is what I ment
